android-sdk/build.gradle

```diff
@@ -30,10 +37,14 @@ dependencies {
 
   compile 'net.danlew:android.joda:2.9.4.2'
   compile 'com.android.support:appcompat-v7:24.2.1'
```

Comment:
AppCompat is at 25 now.  For the appcompat, cardview, preference, design, etc would the best approach to stay relevant be the "+" sign at the end?  

Also, I was wondering if you knew the answer to this question, if i use this library with appcompat v24, but my app.gradle uses v25, will v25 get compiled in or v24?